tests/apps/actorfeatures/.gitignore

```diff
@@ -0,0 +1,3 @@
+app
+actorfeatures
+*.exe
```

Comment:
nit: can we place `.gitignore` in ./tests/apps to share it for all test apps ?

Reply:
sure.